.github/workflows/ci.yml

```diff
@@ -22,54 +22,43 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.runner || 'ubuntu-22.04' }}
-    container: ${{ matrix.container }}
-    name: ${{ matrix.name }}
     strategy:
       fail-fast: false
       matrix:
+        lib_type: [shared, static]
+        # Can't test DB2 as required db2exc package is no longer available after Ubuntu 14.04
+        backend: [sqlite3, postgresql, mysql, firebird, oracle, odbc, empty, valgrind]
+        runner: [ubuntu-22.04]
+        cxxstd: [14]
+        test_release_package: [false]
+        build_examples: [false]
         include:
-          # Note: the jobs are ordered in the order of decreasing running
-          # time, as this should minimize the total run-time of all jobs.
-          - backend: postgresql
+          - backend: empty
             runner: macos-12
-            name: PostgreSQL macOS
-          - backend: oracle
-            name: Oracle 11
             no_boost: true
-          - backend: valgrind
-            name: Valgrind
-          - backend: odbc
-            # There are many leak reports under Ubuntu 22.04, see #1008.
-            container: ubuntu:18.04
-            name: ODBC
-          - backend: firebird
-            name: Firebird
           - backend: postgresql
-            name: PostgreSQL Linux
-          - backend: mysql
-            name: MySQL
-          - backend: sqlite3
             runner: macos-12
-            name: SQLite3 macOS
-          - backend: sqlite3
-            name: SQLite3 C++17
-            cxxstd: 17
+            no_boost: true
           - backend: sqlite3
-            name: SQLite3
-          - backend: empty
             runner: macos-12
-            name: Empty macOS
-          - backend: empty
-            name: Empty
+            no_boost: true
+          - backend: oracle
+            no_boost: true
+            runner: ubuntu-22.04
```

Comment:
If we leave it at the default value, this will effectively be auto-updating. Imo this is not great as this can lead to CI suddenly failing for a PR simply because that PR happened to be the first when GitHub changed the default runner and there is an issue on that specific OS. That would yield to a false negative in the CI (as the failure has nothing to do with the PR).

However, if you actually want this auto-updating behavior, I'll adapt the workflow accordingly.

Reply:
Sorry, I don't remember why did I think that it was redundant any more. If it isn't, let's keep it, I agree that fixing the version is preferable.